uli-community/lib/uli_community_web/live/crowdsource_contributions_live.html.heex

```diff
@@ -125,35 +125,63 @@
   </.simple_form>
 </.modal>
 
-<.header class="text-center">
+<.header class="text-center mb-4">
   Crowdsource Contributions
 </.header>
 
-<.table id="crowdsource-slur-metadata-table" rows={@slurs_metadata_list}>
-  <:col :let={slur} label="Slur's Label"><%= slur.label %></:col>
-  <:col :let={slur} label="Level of Severity"><%= slur.level_of_severity || "-" %></:col>
-  <:col :let={slur} label="Is Casual?">
-    <%= if is_nil(slur.casual), do: "-", else: (slur.casual == true && "Yes") || "No" %>
-  </:col>
+<table class="min-w-full divide-y divide-gray-200 text-center">
```

Comment:
@Mahimahto try to use the tags from phoenix core_components as much as possible. So use `<.table>` and `<:col>` as opposed to `<table>` `<thead>` etc.